src/main/resources/config/application-test.properties

```diff
@@ -42,15 +42,16 @@ spring.thymeleaf.content-type=text/html
 
 # The date format we use
 spring.jackson.date-format=yyyy-MM-dd
+spring.jackson.time-zone=GMT+1
```

Comment:
Did you check if this is the correct timezone, which also deals correctly with daylight savings?